code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -198,6 +198,16 @@
 /obj/item/weapon/storage/box/empshells/large
 	starts_with = list(/obj/item/ammo_casing/a12g/emp = 16)
 
+/obj/item/weapon/storage/box/flechetteshells
+	name = "box of shotgun flechettes"
+	desc = "It has a picture of a gun and several warning symbols on the front.<br>WARNING: Live ammunition. Misuse may result in serious injury or death."
+	icon_state = "lethalslug_box"
+	item_state_slots = list(slot_r_hand_str = "syringe_kit", slot_l_hand_str = "syringe_kit")
+	starts_with = list(/obj/item/ammo_casing/a12g/flechette = 8)
+
+/obj/item/weapon/storage/box/flechetteshells/large
+	starts_with = list(/obj/item/ammo_casing/a12g/flechette = 16)
+

```

Comment:
What's all the above about relative to mechs?

Reply:
It's to help counter Heavy Armor on Exos, yes.